compiler/rustc_codegen_cranelift/build_system/main.rs

```diff
@@ -134,8 +134,7 @@ fn main() {
         }
     }
 
-    let current_dir = std::env::current_dir().unwrap();
-    out_dir = current_dir.join(out_dir);
+    out_dir = rustc_fs_util::try_canonicalize(out_dir).unwrap();
```

Comment:
This won't work here. Rustc_codegen_cranelift's build systen doesn't and can't depend on any rustc crate.

Reply:
Thanks, and indeed `current_dir` is used below. Removed this commit.